### PR TITLE
Updated with Impact Hub chain details

### PR DESCRIPTION
**Pull request by franono**

### CODE REVIEW

**Comment by franono** [comment]
@migueldingli1997

File: impacthub-1/setup.sh [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+echo "Ixo Chain Installer"
+echo -e "-------------------------\n\n"
+sleep 2
+
+if [ "$USER" != "root" ]; then
+        echo "You must be logged in as root to use this installer!"
+        exit 0;
+fi
+
+echo "Starting IXO based install"
```
Comment (migueldingli1997) [inline impacthub-1/setup.sh:15]:
```suggestion
echo "Starting ixo based install"
```

File: impacthub-1/setup.sh [hunk 2]
```diff
@@ -0,0 +1,153 @@
+#!/bin/bash
+
+GOFILENAME="go1.14.1.linux-amd64.tar.gz"
+GODOWNLOADLINK="https://dl.google.com/go/$GOFILENAME"
+
+echo "Ixo Chain Installer"
```
Comment (migueldingli1997) [inline impacthub-1/setup.sh:6]:
```suggestion
echo "ixo Chain Installer"
```

File: impacthub-1/setup.sh [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+
+cd $HOME && curl -O $GODOWNLOADLINK
+sleep 1
+tar -xvf $HOME/$GOFILENAME
+
+rm $HOME/$GOFILENAME
+mv $HOME/go /usr/local
+
+USERNAME=ixo
+echo "Creating new IXO system user"
```
Comment (migueldingli1997) [inline impacthub-1/setup.sh:33]:
```suggestion
echo "Creating new ixo system user"
```

**Review by migueldingli1997 — CHANGES_REQUESTED** [review]
IXO -> ixo

**Review by migueldingli1997 — APPROVED** [review]

**Pull request merged**